src/ansys/fluent/core/services/field_data.py

```diff
@@ -1221,12 +1234,56 @@ def get_scalar_field_data(
                 for surface_id in surface_ids
             }
 
+    @deprecate_argument(
+        old_arg="surface_name",
+        new_arg="surface_names",
+        converter=lambda old_arg_val: [old_arg_val] if old_arg_val else None,
+        deprecation_class=PyFluentDeprecationWarning,
+    )
+    @deprecate_argument(
+        old_arg="data_type",
+        new_arg="provide_vertices",
+        converter=lambda old_arg_val: (
+            True if old_arg_val == SurfaceDataType.Vertices else False
```

Comment:
just `lambda old_arg_val: old_arg_val == SurfaceDataType.Vertices` will be sufficient